test/y2storage/partition_tables/dasd_test.rb

```diff
@@ -76,4 +76,18 @@
       expect(subject.partition_id_supported?(0)).to eq false
     end
   end
+
+  describe "#:supported_partition_ids" do
+    it "for dasd, list includes the LINUX id" do
```

Comment:
NP: Not sure if it makes much sense to repeat "for dasd" in every description of the tests of `PartitionTables::Dasd`

Reply:
rspec looks more readable for me

Reply:
I'd also have to change it in `partition_id_supported?`, consequently